shared/fs/common/path-item-info.js

```diff
@@ -57,20 +57,19 @@ const PathItemInfo = (props: Props) => (
     {props.lastWriter ? (
       <Text type="BodySmall" style={writerTextStyle} lineClamp={isMobile ? 1 : undefined}>
         &nbsp;by&nbsp;
-        <Text type="BodySmall" style={writerStyle}>
-          {props.lastWriter}
-        </Text>
+        <ConnectedUsernames
```

Comment:
can you add `underline={true}`?

Reply:
The design doesn't have an underline.

Reply:
I don't know if that's what Danny is referring to but it should be underlined on hover on desktop.

Reply:
Understood, thanks @cecileboucheron!